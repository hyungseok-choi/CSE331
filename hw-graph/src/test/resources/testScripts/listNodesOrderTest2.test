# Create a graph
CreateGraph graph1

# Add a pair of nodes
AddNode graph1 b
AddNode graph1 a

# Print the nodes in the graph
ListNodes graph1