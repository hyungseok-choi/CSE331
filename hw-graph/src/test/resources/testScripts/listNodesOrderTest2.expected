# Create a graph
CreateGraph graph1

# Add a pair of nodes
added node b to graph1
added node a to graph1

# Print the nodes in the graph
graph1 contains: a b